### PR TITLE
setxkbmap: add complex example and available values examples

### PR DESCRIPTION
**Pull request by PrimaMateria**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by PrimaMateria** [comment]
Hi @schneiderl 
yes, I thought that this is quite obvious command. But on the second thought `-h` and `--help` don't work. Should I put it back?

**Comment by schneiderl** [comment]
@PrimaMateria I agree with you that it is obvious but given that `-h` and `--help` don't work I say we put it back.

File: pages/linux/setxkbmap.md [hunk 1]
```diff
@@ -6,6 +6,22 @@
 
 `setxkbmap {{fr}}`
 
+- Set multiple keyboard layouts, their variants and switching option:
+
+`setxkbmap -layout us,de -variant ,qwerty -option 'grp:alt_caps_toggle'`
```
Comment (sbrl) [inline pages/linux/setxkbmap.md]: We should wrap the bits that users can change in `{{` and `}}`.

Comment (PrimaMateria) [reply pages/linux/setxkbmap.md]: I added curly braces to all arguments that makes sense for user to change. But I was bit unsure if I need to set it to all changeable values or not. For example `grep :grp` or `grep {{:grp}}` in the last example.

Comment (sbrl) [reply pages/linux/setxkbmap.md]: Cool, thanks!

Really, it's only the values that makes sense for the user to change in the contact of your command example.

File: pages/linux/setxkbmap.md [hunk 2]
```diff
@@ -6,6 +6,22 @@
...
+
+`setxkbmap -layout {{us,de}} -variant {{,qwerty}} -option {{'grp:alt_caps_toggle'}}`
+
 - Get help:
 
 `setxkbmap -help`
+
+- List all layouts:
+
+`localectl list-x11-keymap-layouts`
```
Comment (agnivade) [inline pages/linux/setxkbmap.md]: These seem to be a different command. Is this fine ?

**Pull request merged**